src/components/axes/numericAxis.ts

```diff
@@ -1,4 +1,4 @@
-//<reference path="../../reference.ts" />
+///<reference path="../../reference.ts" />
```

Comment:
Not really, this seems to be the way Typescript recommends doing this:

http://www.typescriptlang.org/Handbook#modules-splitting-across-files